style.css

```diff
@@ -0,0 +1,379 @@
+html {
+  align-items: center;
+  width: 100%;
+  display: flex;
+}
+
+body {
+  background: #e9e8e8b6;
+  position: absolute;
+  width: 100%;
+  border: 1px;
+  margin: 20px;
+  left: 0;
+  top: 0;
+  align-items: center;
+}
+
+header {
+  width: 375px;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: basline;
+  margin: 20px;
+  padding-left: 0;
+}
+
+header p {
+  color: #6070ff;
+  width: 140px;
+  height: 30px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 29px;
+  line-height: 20px;
+  display: flex;
+  text-align: start;
+}
+
+header i {
+  font-size: 2.2rem;
+  margin-right: 50px;
+}
+
+button a {
+  text-decoration: none;
+}
+
+.headline a {
+  text-decoration: none;
+}
+
+.wraper {
+  background-color: white;
+  width: 380px;
+  border-radius: 0 0 0 80px;
+}
+
+.headline {
+  background-image: url(/header-shapes\ mobile.png);
+  background-size: 375px;
+  background-repeat: no-repeat;
+  background-position: left + 10px;
+  background-position-y: 40px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex start;
+  padding: 114px 24px;
+  gap: 12px;
+  width: 350px;
+  height: 424px;
+  left: calc(45% - 375px / 2);
+  top: 20px;
+  border-radius: 0 0 8px 0;
+}
+
+h1 {
+  width: 300px;
+  height: 150px;
+  margin: 1px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.details {
+  width: 300px;
+  height: 140px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 28px;
+  color: #344563;
+  margin-bottom: 40px;
+  margin-top: -1px;
+}
+
+h4 {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  text-decoration: none;
+  font-style: normal;
+  color: #7f8cff;
+  margin: 0;
+}
+
+.social ul {
+  display: flex;
+  width: 150px;
+  height: 40px;
+  justify-content: space-between;
+  font-size: 30px;
+  list-style: none;
+  margin: 0;
+  padding: 0;
```

Comment:
- [ ] Also noted by the previous reviewer, you'd agree your design is at this point not visually appealing, but it can be improved upon. Kindly stick to the figma design to make it look good from a user view 👍 

## View:
![image](https://user-images.githubusercontent.com/50990925/220543602-873e2b6f-36be-49a2-9280-0ca8652b419f.png)

## Figma
![image](https://user-images.githubusercontent.com/50990925/220543911-c262ad5a-99b1-434d-a43b-dd67f87a4a37.png)